politeiawww/cmd/pictl/print.go

```diff
@@ -52,3 +55,66 @@ func byteCountSI(b int64) string {
 func printInPlace(s string) {
 	fmt.Printf("\033[2K\r%s", s)
 }
+
+// dollars converts an int64 price in cents into a human readable dollar
+// string.
+//
+// | Input     | Output          |
+// |-----------------------------|
+// | 130000000 | "$1,300,000.00" |
+// | 13000000  | "$130,000.00"   |
+// | 130000    | "$1,300.00"     |
+// | 13000     | "$130.00"       |
+// | 78        | "$0.78"         |
+// | 9         | "$0.09"         |
+// | 0         | "$0.00"         |
+// | -9        | "-$0.09"        |
+// | -78       | "-$0.78"        |
+// | -13000000 | "-$130,000.00"  |
+func dollars(cents int64) string {
+	// Get the value in dollars.
+	dollarsValue := float64(cents) / 100
+
+	// Initialize the buffer to store the string result.
+	var buf bytes.Buffer
+
+	// Check for a negative value.
+	if dollarsValue < 0 {
+		buf.WriteString("-")
+		// Convert the negative value to a positive value. The code
+		// below can only handle positive values.
+		dollarsValue = 0 - dollarsValue
+	}
+	buf.WriteString("$")
+
+	// Split the value into integer and decimal.
+	parts := strings.Split(strconv.FormatFloat(dollarsValue, 'f', -1, 64), ".")
+
+	// Process the integer part.
+	var position int
+	var integerPart = parts[0]
+
+	// Because the number of integer is divided per thousand(3 letters).
```

Comment:
These comments are better.  It's easier to understand the intent of your code now.